### DISCUSSION TITLE
How to prevent ComfyUI from automatically downloading SD 1.5?

### DISCUSSION THREAD

**Discussion by BlinkerHigh** [Comfy-Org/ComfyUI#4332] [Q&A] [answered]
I wanted to get rid of the model, but ComfyUI automatically downloads it, saying it is required.

![image](https://github.com/user-attachments/assets/62f011db-a660-4064-85d3-e4de3b69848c)

**Comment by ltdrdata** [comment, marked as answer]
ComfyUI never downloads the model automatically.
Your issue is occurring outside of ComfyUI.
